bitnami/airflow/README.md

```diff
@@ -976,6 +976,41 @@ The Bitnami Airflow chart relies on the PostgreSQL chart persistence. This means
 | `worker.networkPolicy.ingressNSMatchLabels`                | Labels to match to allow traffic from other namespaces                                                                                                                                                                          | `{}`             |
 | `worker.networkPolicy.ingressNSPodMatchLabels`             | Pod labels to match to allow traffic from other namespaces                                                                                                                                                                      | `{}`             |
 
+### Airflow "setup-db" K8s Job parameters
```

Comment:
I'm missing a documentation section explaining the two options

Reply:
Done at https://github.com/bitnami/charts/pull/30440/commits/f15f2ec29d0f2ef072e0257a61af260754cc7406